frontend/src/lib/components/TaxonomicFilter/infiniteListLogic.ts

```diff
@@ -195,6 +198,20 @@ export const infiniteListLogic = kea<infiniteListLogicType>([
         ],
     })),
     reducers(({ props }) => ({
+        intelligentQuery: [
+            '',
+            {
+                setSearchQuery: (_, { searchQuery }) => {
+                    if (props.listGroupType === TaxonomicFilterGroupType.EventProperties && isURL(searchQuery)) {
+                        return '$current_url'
+                    }
+                    if (props.listGroupType === TaxonomicFilterGroupType.PersonProperties && isEmail(searchQuery)) {
+                        return 'email' // TODO not everyone will call this email 🤷
```

Comment:
This will cause problems (like that user reported in the Loom) whereby we add suggested properties that do not exist on their account e.g. Email vs email

Reply:
yep, this is the figure out where the code would have to go version 🤣 

what do you think of the overall interaction? confusing to paste a URL and not search for it?

Reply:
Nope, I think it's intuitive. I guess if someone types a URL today no results are returned so it's a bit of a dead end - at least now we can guide them to what they are most likely looking for :)